src/components/panel/index.njk

```diff
@@ -21,6 +21,42 @@ Panels are used to draw information to the attention of the user.
     patternlibExample({"path": "components/panel/examples/warn/index.njk"})
 }}
 
+### Warning with countdown
+{{
+   patternlibExample({"path": "components/timeout-panel/examples/warn/index.njk"})
+}}
+
+Use this warning panel variant when a service sets a session expiry time to protect the user’s information.
+
+When the countdown reaches zero, the browser will navigate to a specified URL and show some alternate text. If the current URL is provided the page will just be refreshed.
```

Comment:
Is "show some alternate text" referring to the message shown in the panel when the time runs out? If so, that should come first...
```suggestion
When the session expiry timer reaches zero, the `countdownExpiredText` message will be displayed and the browser will redirect to the URL set for a [session timed out error page](/patterns/error-status-pages/#session-timed-out-error-when-user-is-not-signed-in). If the current URL is set, the page will reload.
```